apps/SageAccountWeb/src/svg.d.ts

```diff
@@ -4,7 +4,4 @@ declare module '*.svg' {
   export const ReactComponent: React.FunctionComponent<
     React.SVGProps<SVGSVGElement> & { title?: string }
   >
-
-  const dataUrl: string
-  export default dataUrl
```

Comment:
This type of svg import no longer works (you need to import from `'path/to/foo.svg?url'`, so remove it from our custom type definitions